libs/core/src/lib/formatted-text/formatted-text.component.ts

```diff
@@ -0,0 +1,101 @@
+import {
+    Component,
+    ViewEncapsulation,
+    ChangeDetectionStrategy,
+    Input,
+    OnChanges,
+    HostBinding,
+    OnInit
+} from '@angular/core';
+import { DomSanitizer, SafeHtml } from '@angular/platform-browser';
+
+import { HtmlSanitizer } from './utils/html-sanitizer';
+
+export type LinkTargetType = '' | '_blank' | '_self' | '_top' | '_parent' | '_search';
+
+/**
+ * Formatted-text allowed tags, only this tags will render in component, other will skip;
+ *
+ * ``` selector: fd-formatted-text ```
+ *
+ * ```html
+ * <fd-formatted-text [htmlText]="..."></fd-formatted-text>
+ * ```
+ */
+
+@Component({
+    selector: 'fd-formatted-text',
+    exportAs: 'fd-formatted-text',
+    template: ``,
+    styles: [
+        `
+            fd-formatted-text {
+                display: block;
+            }
+            fd-formatted-text.fd-formatted-text-with-height {
+                overflow-y: auto;
+            }
+            fd-formatted-text.fd-formatted-text-with-width {
+                overflow-x: auto;
+            }
+        `
```

Comment:
It's a very simple style for this component, and won't append more. Are you sure that we need to create an extra style file?

Reply:
I think it would be cleaner, in the separate file we can use preprocessors and have all advantages of IDE (intellisense and so on)

Reply:
As for me, it's more readable. Also, it's the easiest way to add a short style for your component, it will be enough and it will never increase. In this case, the component will be self-sufficient.
Sorry, but `all advantages of IDE` is not a great reason to create a separate file.